.prettierignore

```diff
@@ -17,3 +17,6 @@ build
 
 # dependencies
 node_modules
+
+# other (avoids breaking a link)
+app/components/common/MDXContent/hca-dcp/downloadBatchCorrectionWarning.mdx
```

Comment:
I added this since Prettier was splitting `({portalURL}/guides/consumer-vignettes/matrices#matrix-normalization-and-batch-correction)` onto two lines, causing an error building the page. Not sure if there's a better way to handle this